pupa/tests/importers/test_vote_event_importer.py

```diff
@@ -375,3 +376,39 @@ def test_vote_event_bill_actions_errors():
     # these both try to match the same action, only first will succeed
     assert votes[2].bill_action is not None
     assert votes[3].bill_action is None
+
+
+@pytest.mark.django_db
```

Comment:
this ensures fix_bill_id-type behavior is possible to implement still

the test on VoteEvent demonstrates part of why this has to be done at import time